### DISCUSSION TITLE
Renovate opening and closing PRs automatically causing explosion in build runs

### DISCUSSION THREAD

**Discussion by graemerocher** [renovatebot/renovate#22572] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Describe the problem

Micronaut is a popular Open Source Java framework with a Gradle based build infrastructure. It is built using a multi repository approach and renovate is enabled globally on an organization level.

Recently Renovate started opening and closing PRs for different repositories in the organization every few hours causing our build runners to be constantly occupied. It is happening across our organization on all repositories but here is an example PR

https://github.com/micronaut-projects/micronaut-security/pull/1338

The logs for when the PR is opened and auto closed are attached as gists since they were too large to include.

There is nothing clear from the logs as to why this is happening unless I am missing something? Is this a bug?

### Relevant debug logs

<details><summary>Auto Close Logs</summary>
https://gist.github.com/graemerocher/7a19aca4495d27fe7de1707b96a05719
</details>

<details><summary>Open PR Logs</summary>
https://gist.github.com/graemerocher/37a4d54f9cda974aa5c7a6c2a843a6bd
</details>

### Have you created a minimal reproduction repository?

I have linked to a minimal reproduction in the description above

**Comment by rarkins** [comment]
One the autoclosing run, the branch is `renovate/micronaut.logging` while when it's opened, the branch is `renovate/micronaut-logging`

The first case has:
```
            "depName": "io.micronaut.logging:micronaut-logging-bom",
            "groupName": "micronaut.logging",
```

The second has:

```
           "depName": "io.micronaut.logging:micronaut-logging-bom",
            "groupName": "micronaut-logging",
```

The second probably comes from the variable name https://github.com/micronaut-projects/micronaut-security/blob/57ce46916ddc7122161d577a3333f38df91bd241/gradle/libs.versions.toml#L30

I'm not sure about the first, or why it would be inconsistent.

@Churro any ideas?

**Reply by Churro** [reply]
PR #22470 (-> renovate 35.104.0) aligned the normalization scheme to the one used by gradle, hence `"micronaut.logging"` is the current group name and `"micronaut-logging"` the one prior to that PR. When running renovate locally on that repo, I expectedly get `micronaut.logging` consistently.

I suspect the issue is that two renovate instances are working "against" each other:
* Open PR log: `"renovateVersion": "35.102.10"`
* Auto-close log: `"renovateVersion": "35.105.2"`

Could it be an issue with the orchestration of renovate runners?
From the `currentHourStart` field I understand that there are 2 hours between these two runs.

**Reply by nabeelsaabna** [reply]
Thanks for the detailed info, I did find few runners failed to update from `35.102.10` to `35.105.2`

**Reply by graemerocher** [reply]
Thanks, do you know when they will be correctly updated? As of this morning we still have all our build runners pegged by renovate PRs

**Reply by nabeelsaabna** [reply, marked as answer]
Updating all runners to `renovate:35.110.0`  is in progress

**Reply by graemerocher** [reply]
Thank you!
